pyproject.toml

```diff
@@ -64,6 +64,7 @@ dependencies = [
     "jsonpath-ng>=1.6.1, <2",
     "Mako>=1.2.2",
     "markdown>=3.0",
+    "marshmallow<4",
```

Comment:
Let's always put a comment above when setting an upper bound that's not based on pure semver assumptions, something like "marshmallow>=4 has shows issues related to `'minLength'`", ...

Reply:
Thank you. I didn't see such comments on other spots, but I've just added it, effectively linking to the corresponding issue on GitHub.